### PR TITLE
Fixed some grammar issues and typos

### PR DESCRIPTION
**Pull request by AndrewDawes**
I found and fixed some typos, grammar issues and made some wording sound a bit more natural :-)

### CODE REVIEW

**Comment by jmooring** [comment]
Files in content/en/commands are generated from source (https://github.com/gohugoio/hugo). You would have to make changes to the related .go files, compile, run `hugo gen docs`, replace the files in docs/content/en/commands, then submit a PR.

Please remove the command files from this PR.

**Comment by AndrewDawes** [comment]
@jmooring  Ah ok, my mistake. Command files removed.

File: content/en/content-management/sections.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Also see [Page Variables](/variables/page/).
 
 ## Content Section Lists
 
-Hugo will automatically create pages for each *root section* that list all of the content in that section. See the documentation on [section templates][] for details on customizing the way these pages are rendered.
+Hugo will automatically create pages for each *root section* that list all the content in that section. See the documentation on [section templates][] for details on customizing the way these pages are rendered.
```
Comment (jmooring) [inline content/en/content-management/sections.md:90]:
```suggestion
Hugo will automatically create a page for each *root section* that lists all the content in that section. See the documentation on [section templates][] for details on customizing the way these pages are rendered.
```

File: content/en/content-management/shortcodes.md [hunk 2]
```diff
@@ -176,7 +176,7 @@ If the gist contains several files and you want to quote just one of them, you c
 
 #### Example `gist` Display
 
-To demonstrate the remarkably efficiency of Hugo's shortcode feature, we have embedded the `spf13` `gist` example in this page. The following simulates the experience for visitors to your website. Naturally, the final display will be contingent on your stylesheets and surrounding markup.
+To demonstrate the remarkable efficiency of Hugo's shortcode feature, we have embedded the `spf13` `gist` example in this page. The following simulates the experience for visitors to your website. Naturally, the final display will be contingent on your stylesheets and surrounding markup.
```
Comment (jmooring) [inline content/en/content-management/shortcodes.md:179]:
```suggestion
To demonstrate the remarkable efficiency of Hugo's shortcode feature, we have embedded the `spf13` `gist` example in this page. The following simulates the experience for visitors to your website. Naturally, the final display will depend on your stylesheets and surrounding markup.
```

File: content/en/getting-started/configuration.md [hunk 3]
```diff
@@ -508,7 +508,7 @@ useResourceCacheWhen
 writeStats {{< new-in "0.69.0" >}}
 : When enabled, a file named `hugo_stats.json` will be written to your project root with some aggregated data about the build, e.g. list of HTML entities published to be used to do [CSS pruning](/hugo-pipes/postprocess/#css-purging-with-postcss). If you're only using this for the production build, you should consider placing it below [config/production](/getting-started/configuration/#configuration-directory). It's also worth mentioning that, due to the nature of the partial server builds, new HTML entities will be added when you add or change them while the server is running, but the old values will not be removed until you restart the server or run a regular `hugo` build.
 
-**Note** that the prime use case for this is purging of unused CSS; it is build for speed and there may be false positives (e.g. elements that isn't really a HTML element).
+**Note** that the prime use case for this is purging of unused CSS; it is build for speed and there may be false positives (e.g. elements that isn't really an HTML element).
```
Comment (jmooring) [inline content/en/getting-started/configuration.md:511]:
```suggestion
**Note** that the prime use case for this is purging of unused CSS; it is built for speed and there may be false positives (e.g., detection of HTML elements that are not HTML elements).
```

File: content/en/hosting-and-deployment/hosting-on-netlify.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ Select the repo you want to use for continuous deployment. If you have a large n
 
 ![Screenshot of step 1 of create a new site for Netlify: selecting the git provider](/images/hosting-and-deployment/hosting-on-netlify/netlify-create-new-site-step-2.jpg)
 
-Once selected, you'll be brought to a screen for basic setup. Here you can select the branch you wanted published, your [build command][], and your publish (i.e. deploy) directory. The publish directory should mirror that of what you've set in your [site configuration][config], the default of which is `public`. The following steps assume you are publishing from the `master` branch.
+Once selected, you'll be brought to a screen for basic setup. Here you can select the branch you want to be published, your [build command][], and your publish (i.e. deploy) directory. The publish directory should mirror that of what you've set in your [site configuration][config], the default of which is `public`. The following steps assume you are publishing from the `master` branch.
```
Comment (jmooring) [inline content/en/hosting-and-deployment/hosting-on-netlify.md:58]:
```suggestion
Once selected, you'll be brought to a screen for basic setup. Here you can select the branch you want to publish, your [build command][], and your publish (i.e. deploy) directory. The publish directory should mirror that of what you've set in your [site configuration][config], the default of which is `public`. The following steps assume you are publishing from the `master` branch.
```

File: content/en/templates/menu-templates.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ toc: false
 ---
 
 Hugo makes no assumptions about how your rendered HTML will be
-structured. Instead, it provides all of the functions you will need to be
+structured. Instead, it provides all the functions you will need to be
```
Comment (jmooring) [inline content/en/templates/menu-templates.md:23]:
```suggestion
structured. Instead, it provides all the functions you will need to
```

File: content/en/templates/menu-templates.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ toc: false
...
 able to build your menu however you want.
```
Comment (jmooring) [inline content/en/templates/menu-templates.md:24]:
```suggestion
build your menu however you want.
```

File: content/en/templates/shortcode-templates.md [hunk 7]
```diff
@@ -100,7 +100,7 @@ most helpful when the condition depends on either of the values, or both:
 
 #### `.Inner`
 
-If a closing shortcode is used, the `.Inner` variable will be populated with all of the content between the opening and closing shortcodes. If a closing shortcode is required, you can check the length of `.Inner` as an indicator of its existence.
+If a closing shortcode is used, the `.Inner` variable will be populated with all the content between the opening and closing shortcodes. If a closing shortcode is required, you can check the length of `.Inner` as an indicator of its existence.
```
Comment (jmooring) [inline content/en/templates/shortcode-templates.md:103]:
```suggestion
If a closing shortcode is used, the `.Inner` variable will be populated with the content between the opening and closing shortcodes. If a closing shortcode is required, you can check the length of `.Inner` as an indicator of its existence.
```

**Review by jmooring — CHANGES_REQUESTED** [review]

**Pull request merged**